internal/cmd/ske/cluster/create/create.go

```diff
@@ -87,13 +87,15 @@ func NewCmd(p *print.Printer) *cobra.Command {
 				}
 			}
 
-			// Check if SKE is enabled for this project
+			// Check if the project is enabled before trying to create
 			enabled, err := skeUtils.ProjectEnabled(ctx, apiClient, model.ProjectId)
 			if err != nil {
 				return err
 			}
 			if !enabled {
-				return fmt.Errorf("SKE isn't enabled for this project, please run 'stackit ske enable'")
+				return &errors.ServiceDisabledError{
+					Service: "ske",
```

Comment:
```suggestion
					Service: "SKE",
```

Reply:
I think we are using the capitalized `SKE` everywhere

Reply:
This is just used for the command suggestion to enable (as `stackit %s enable`), similarly to what we do in other errors